libs/build_dod_dataset.py

```diff
@@ -49,145 +52,82 @@ def _get_abbrev_df():
-    # assert unique key test
-    assert final_df["Combined Key"].value_counts().max() == 1
-    assert final_df["State/County FIPS Code"].value_counts().max() == 1
+    data = JHUDataset.local().timeseries()
+    data = data.get_subset(None, country="USA")
+    return data.latest_values(AggregationLevel.COUNTY)
+
 
-    return final_df
+def _get_usa_by_states_df():
+    data = JHUDataset.local().timeseries()
+    data = data.get_subset(None, country="USA")
+    return data.latest_values(AggregationLevel.STATE)
 
 
 def get_usa_by_county_with_projection_df(input_dir, intervention_type):
     us_only = _get_usa_by_county_df()
-    fips_df = FIPSPopulation.local().data  # used to get interventions
-    interventions_df = (
-        _get_interventions_df()
-    )  # used to say what state has what interventions
+    fips_df = FIPSPopulation.local().data
+    # used to get interventions
+    interventions_df = _get_interventions_df()
     projections_df = get_county_projections_df(
         input_dir, intervention_type, interventions_df
     )
 
     counties_decorated = (
         us_only.merge(
             projections_df,
-            left_on="State/County FIPS Code",
+            left_on=TimeseriesDataset.Fields.FIPS,
             right_on="FIPS",
             how="inner",
         )
-        .merge(fips_df[["state", "fips"]], left_on="FIPS", right_on="fips", how="inner")
         .merge(interventions_df, left_on="state", right_on="state", how="inner")
     )
-
     counties_remapped = counties_decorated.rename(
         columns=OUTPUT_COLUMN_REMAP_TO_RESULT_DATA
     )
     counties = pd.DataFrame(counties_remapped, columns=RESULT_DATA_COLUMNS_COUNTIES)
     counties = counties.fillna(NULL_VALUE)
     counties.index.name = "OBJECTID"
+    counties["Province/State"] = counties["Province/State"].map(abbrev_us_state)
+
     # assert unique key test
+    assert counties["State/County FIPS Code"].value_counts().max() == 1
 
-    assert counties["Combined Key"].value_counts().max() == 1
     return counties
 
 
 def get_usa_by_states_df(input_dir, intervention_type):
-
-    us_only = _get_usa_by_county_df()
-    abbrev_df = _get_abbrev_df()
+    us_only = _get_usa_by_states_df()
     interventions_df = _get_interventions_df()
     projections_df = get_state_projections_df(
         input_dir, intervention_type, interventions_df
     )
-
-    states_group = us_only.groupby(["Province/State"])
-    states_agg = states_group.aggregate(
-        {
-            "Last Update": "max",
-            "Confirmed": "sum",
-            "Recovered": "sum",
-            "Deaths": "sum",
-            "Active": "sum",
-            "Country/Region": "first",
-            "Latitude": "first",
-            "Longitude": "first"
-            # People tested is currently null
-            #'People Tested': 'sum'
-        }
-    )
-
-    # basically the states_agg has full state names, the interventions have abbreviation so we need these to be merged
```

Comment:
why does this change?

Reply:
This is all handled in the Data source/timeseries preprocessing, the datasets coming out of it are already one row per state